### DISCUSSION TITLE
Graphic Computational geometric cut window algorithm

### DISCUSSION THREAD

**Discussion by pcbren1** [pixijs/pixijs#7679] [Q&A]
How can I get it; The enlarged graphic, that is, the graphic list or graphic index of the graphic displayed on the current screen
Does Pixi have such a method
![5454](https://user-images.githubusercontent.com/86771584/128502799-281af0ea-7f23-4877-9614-a9b1c4832e05.png)

Computational geometric cut window algorithm  Example
![20200826155801174](https://user-images.githubusercontent.com/86771584/128503265-7a504c33-029a-4c7e-b638-313d31ed269e.png)

**Comment by ivanpopelyshev** [comment]
No, Pixi doesnt have algorithms like that but i add them in my projects regularly, i dont see a problem. I thought at this point you already read `Container` , `Graphics` sources? Btw, do you use graphics-smooth for that demo?
